### PR TITLE
Fix for PP-262: Add pull request template

### PR DESCRIPTION
**Pull request by hirenvadalia**
#### Issue
* PP-262

#### Description
* This pull adds Pull Request Template to master branch.

#### Steps to Test or Reproduce
* None, because this PR is just to add pull request template

#### Impacted Areas in Application
* None

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#### Issue
+* PP-XXXX
+
+#### Description
+* A few sentences describing the overall goals of the pull request's commits.
+
+#### Steps to Test or Reproduce
```
Comment (mike0042) [inline .github/PULL_REQUEST_TEMPLATE.md]: I thought this information was supposed to be present in the OpenJIRA ticket. Will QA need to reference both the ticket and the pull request? Will this information automatically be added to the ticket?

Comment (hirenvadalia) [inline .github/PULL_REQUEST_TEMPLATE.md]: Yes @mike0042 this information supposed to be present in JIRA ticket. I don't think QA will need to look into both ticket and pull.. QA will look only into JIRA ticket... And this information in pull is for reviewers and collaborators...
No this information will not be added to JIRA automatically.... user has to add manually

**Comment by subhasisb** [comment]
travis was not working after we moved from private to public and so we had to refresh its settings. Closing this pull, so Hiren can reopen and try once more.

**Comment by subhasisb** [comment]
My feedback about the pull request template. I believe what should go in the template should be everything that we usually write in an email to cause of the problem (Jira has the high level problem description, but we can always describe the technical cause of the problem), and the fix solves the problem (solution). 

The pull request is really an notification for others to start reviewing code that we submitted, so the pull request template should help capture details that will help in the code review process. At the same time, it would be helpful if we dont have to duplicate content that is already in JIRA

**Pull request closed without merging**